### PR TITLE
DemandSpike - Add Statistics Gathering from Kafka Random Byte Write #70

### PR DESCRIPTION
**Pull request by louzar**

### CODE REVIEW

File: libs/kafka_2.8.0-0.8.1.jar.asc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+-----BEGIN PGP SIGNATURE-----
```
Comment (smorin) [inline libs/kafka_2.8.0-0.8.1.jar.asc]: Please remove this file form the pull request

**Comment by smorin** [comment]
Before we can commit please update the project to build with http://www.gradle.org/ and use gradle to include all jar dependencies.  Please specify the versions of the jars that it uses.

**Comment by smorin** [comment]
Update the package names from things like: com.demandspike.kafka.producer  to com.demandcube.demandspike.kafka.producer

File: .classpath [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (smorin) [inline .classpath:1]: hidden files shouldn't be checked in

File: .gradle/1.10/taskArtifacts/cache.properties [hunk 3]
```diff
@@ -0,0 +1 @@
+#Wed Apr 23 15:59:11 CET 2014
```
Comment (smorin) [inline .gradle/1.10/taskArtifacts/cache.properties:1]: all the gradle hidden files need to be removed

File: .project [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>DemandSpike</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>
+			<arguments>
```
Comment (smorin) [inline .project:10]: Project file should be removed.

**Comment by smorin** [comment]
Remove the build directory and all build files

**Comment by smorin** [comment]
Add paths that need to be removed to .gitignore

Please in the future for all future projects:
don't checkin jars (unless you can't get them from gradle)
don't checkin build directories
don't checking classes
don't checkin hidden files.

This is bad practice

**Comment by smorin** [comment]
@louzar I haven't seen responses to any of these comments?  Can you please address them all?  Also haven't seen updates to the pull request to fix these problems.  Have you tried to fix them?

**Comment by louzar** [comment]
new pull request : https://github.com/DemandCube/DemandSpike/pull/2

**Comment by smorin** [comment]
This wasn't fixed in this pull request

**Pull request closed without merging**